dev-python/pipenv/Manifest

```diff
@@ -1 +1,2 @@
 DIST pipenv-2018.11.26.tar.gz 6560580 BLAKE2B a3846de696733b5f3f393163999d7f333b52accdcebadfd800507defd1108ac4ccb288b39fb805ecf03e417cfe0d1a8b2fc714a95e8ee4485f8622ff8fe4a1ca SHA512 59582ad4a0730083ea58867f7a5d9714290d62947f56bec80e154d4359fe1dff16e44974a7fba3841e5f2516f2ce9709baf9954c9fc40fd66ebc46c3060d4dea
+DIST pipenv-2020.4.1_beta_p1.tar.gz 11331009 BLAKE2B 55ac81238b885bdf0293199ca1caf44a78f3cdbec75c9df259411b949dec8b385161e7ef46160ca099ef51041410d8c5d1274ad200eba5c2a9d06c8a5f61c5f1 SHA512 a5de90ae9910a30666ac7323ccf83b78ee11d8993aef1a2108a44f4256d365d0c1abfff72c83748ef82215914931090c670a10eeb511cb48483762b54c80e5c7
```

Comment:
The version should be `2020.4.1_beta1` rather than `2020.4.1_beta_p1`.

Reply:
This is fixed.